gettsim/taxes/zve.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 
 
-def zve(tax_unit, tb):
+def zve(tax_unit, params, soz_vers_beitr_params, kindergeld_params):
```

Comment:
Again, maybe use `e_st_abzuege_params` because the groups are not so obvious here.

Reply:
Changed